### PR TITLE
Made the min and max functions

### PR DESCRIPTION
**Pull request by addywanderer**
Hi 😸 😻 🔢 Da first PR of Da great Churro!

### CODE REVIEW

File: math_lib.py [hunk 1]
```diff
@@ -1,9 +1,17 @@
 def min(lst):
-    pass
+    min_so_far = 1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
```
Comment (victoongo) [inline math_lib.py]: what if the actual min of the list is bigger than this constant?

File: math_lib.py [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
+    for i in lst:
+        if i < min_so_far:
+            min_so_far = i
+    return min_so_far
 
 
 def max(lst):
-    pass
+    max_so_far = 0
```
Comment (victoongo) [inline math_lib.py]: what is the max of the list is negative?

**Review by victoongo — CHANGES_REQUESTED** [review]
good job on your first PR. I have added my review and requested some changes. Thanks!

File: math_lib.py (hunk 2 above)
Comment (addywanderer) [reply math_lib.py]: Hmm. I will work on it 👍 👍 👍 👍 👍 👌 🆗

File: math_lib.py (hunk 1 above)
Comment (addywanderer) [reply math_lib.py]: Good suggestion! Problem adressed. 👍 😃

**Review by victoongo — APPROVED** [review]

**Pull request merged**